src/app/about/page.tsx

```diff
@@ -1,26 +1,30 @@
 import Image from "next/image";
-import HeroImage from "public/hero.jpg";
-import LinkBtn from "@/components/LinkBtn";
+import heroImage from "public/hero.jpg";
```

Comment:
Capitalized version is better, as this variable is a image stacic constant

```suggestion
import HeroImage from "public/hero.jpg";
```